src/asela/input/default.json

```diff
@@ -23,7 +23,7 @@
     "eps_time"   : 1e-9, //( stop for time inversion) 
     //----------------------Physical parameters----------------------------
     "mu"          : -0.000544617, //(-m_e/m_i) -0.000544617, -0.000272121, -0.000181372
-    "tau"         : 1,      //T_i/T_e
+    "tau"         : 0,      //T_i/T_e
     "beta"        : 5e-6,      //Plasma beta 5e-6 tjk, 4e-3 compass
     "nu_perp"     : 1e-3,   //  (perpendicular viscosity)
     "nu_parallel" : 1,     //  (parallel viscosity)
```

Comment:
Eine Frage wäre, ob wir die comments aus den json files rausnehmen wollen, weil ja in Zukunft alles im pdf file stehen soll? Beim toefl hab ich sie schon draußen.

Reply:
erst wenn die doku komplett ist. Und zum entwickeln sind sie ja schon nützlich

Reply:
ja, genau, wenns noch kein tex file gibt sollten die Kommtare schon noch im json file drin bleiben. Wenns aber ein tex file gibt, sollte  meiner Meinung nach das tex file die Referenz sein und nicht das json file. Was, wenn z.B. im tex und json file unterschiedliche Kommentare stehen? Wenn man die doppelt hat muss man auch immer an zwei Stellen anpassen wenn man was verändert. Ich bin da eher für Kommentare rausnehmen, aber im Prinzip kann das auch jeder selber für sein Projekt entscheiden.